scipy/stats/stats.py

```diff
@@ -1805,6 +1810,18 @@ def cumfreq(a, numbins=10, defaultreallimits=None, weights=None):
     >>> extrapoints
     3
 
+    Generate histogram of cumulative frequencies
```

Comment:
This needs a blank line between desciption and doctest-format code, or it won't render correctly.